pyvista/plotting/plotting.py

```diff
@@ -1300,7 +1300,9 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
         ##### Parse arguments to be used for all meshes #####
 
         if scalar_bar_args is None:
-            scalar_bar_args = {'n_colors': n_colors}
+            scalar_bar_args_copy = {'n_colors': n_colors}
+        else:
+            scalar_bar_args_copy = scalar_bar_args.copy()
```

Comment:
I don't think we should rename the kwarg. The only sane behaviour is the copy, so just doing
```py
scalar_bar_args = scalar_bar_args.copy()
```
should suffice in my opinion. With perhaps a comment to make it clear that we're doing this so as not to mutate the input.

Reply:
Much better idea.  The diff is smaller too